openapi/spec/openapi.json

```diff
@@ -2381,6 +2420,95 @@
       },
+      "Broadcast": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type": "string",
+            "description": "The unique id for this broadcast.",
+            "readOnly": true
+          },
+          "title": {
+            "type": "string",
+            "description": "Title of the broadcast.",
+            "nullable": false,
+            "maxLength": 255
+          },
+          "content": {
+            "type": "string",
+            "description": "Content of the broadcast.",
+            "nullable": true,
+            "maxLength": 4194304
+          },
+          "action_url": {
+            "type": "string",
+            "format": "uri",
+            "description": "A URL to redirect the user to when they click the notification in their notification inbox.",
+            "nullable": true,
+            "maxLength": 2048
+          },
+
+          "category": {
+            "type": "string",
+            "description": "Category the notification belongs to. This is useful to allow users to set their preferences.",
+            "nullable": true,
+            "maxLength": 100
+          },
+          "topic": {
+            "type": "string",
+            "description": "Topic the notification belongs to. This is useful to create threads.",
+            "nullable": true,
+            "maxLength": 100
+          },
+          "custom_attributes": {
+            "type": "object",
+            "description": "Set of key-value pairs that you can attach to a notification, 6KB at maximum. It accepts objects for the value of a key.\n\nYou can use it to share data between channels or to render a custom UI.",
+            "nullable": true,
+            "additionalProperties": true
+          },
+          "sent_at": {
+            "type": "number",
+            "description": "The timestamp when the notification was sent to its recipients.",
+            "readOnly": true
+          },
+          "recipients": {
+            "type": "array",
+            "items": {
+              "anyOf": [
+                { "$ref": "#/components/schemas/MatchesRecipient" },
+                { "$ref": "#/components/schemas/EmailRecipient" },
+                { "$ref": "#/components/schemas/ExternalIdRecipient" }
+              ]
+            },
+            "nullable": false,
+            "minItems": 1,
+            "maxItems": 1000
+          },
+          "recipients_count": {
+            "type": "integer",
+            "description": "The number of recipients that the broadcast was sent to.",
+            "readOnly": true,
+            "nullable": false
+          },
+          "overrides": {
+            "type": "object",
+            "additionalProperties": true
+          },
+          "processing_status": {
+            "status": "string",
+            "summary": {
+              "type": "object",
+              "additionalProperties": false,
```

Comment:
This can contain `errors`, which can technically contain more fields. Much like imports.

Reply:
Thanks! Forgot the `errors` property.